library-backend/library-app/src/test/java/org/internship/library/app/controller/BookControllerIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+package org.internship.library.app.controller;
+
+import org.internship.library.api.Book;
+import org.internship.library.app.LibraryAppConfigTest;
+import org.internship.library.app.persistence.entity.BookEntity;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * Integration Testing Class from Controller to DB
+ */
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {LibraryAppConfigTest.class})
+public class BookControllerIntegrationTest {
+
+    TestRestTemplate testRestTemplate = new TestRestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+
+    /**
+     * Verify if get request return a book from database
+     */
+    @Test
+    public void getBook() {
+
+        String uri = "http://localhost:7777/library/book/{id}";
+        final String testBookId = "5";
+        ResponseEntity responseEntity = testRestTemplate.getForEntity(uri, BookEntity.class, testBookId);
+        assertNotNull(responseEntity);
+        assertEquals(responseEntity.getStatusCode(), HttpStatus.OK);
```

Comment:
You should also verify that if you've requested book with id 5, the id of the book retrieved is 5 and not some other id.